### PR TITLE
Compute V2: add Quotaset resource

### PR DESCRIPTION
**Pull request by ozerovandrei**
Add "resource_openstack_compute_quotaset_v2" with acceptance tests.

For #657

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -67,6 +67,7 @@ github.com/googleapis/gax-go/v2 v2.0.4/go.mod h1:0Wqv26UfaUD9n4G6kQubkQ+KchISgw+
 github.com/googleapis/gax-go/v2 v2.0.5 h1:sjZBwGj9Jlw33ImPtvFviGYvseOtDM7hkSKB7+Tv3SM=
 github.com/googleapis/gax-go/v2 v2.0.5/go.mod h1:DWXyrwAJ9X0FpwwEdw+IPEYBICEFu5mhpdKc/us6bOk=
 github.com/gophercloud/gophercloud v0.0.0-20190212181753-892256c46858/go.mod h1:vxM41WHh5uqHVBMZHzuwNOHh8XEoIEcSTewFxm1c5g8=
+github.com/gophercloud/gophercloud v0.6.0 h1:Xb2lcqZtml1XjgYZxbeayEemq7ASbeTp09m36gQFpEU=
```
Comment (ozerovandrei) [inline go.sum]: This is strange.

File: openstack/resource_openstack_compute_quotaset_v2.go [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/quotasets"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceComputeQuotasetV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeQuotasetV2Create,
+		Read:   resourceComputeQuotasetV2Read,
+		Update: resourceComputeQuotasetV2Update,
+		Delete: resourceComputeQuotasetV2Delete,
```
Comment (kayrus) [inline openstack/resource_openstack_compute_quotaset_v2.go]: you can also use this func:
```suggestion
		Delete: schema.RemoveFromState,
```

Comment (ozerovandrei) [reply openstack/resource_openstack_compute_quotaset_v2.go]: Nice catch

**Comment by ozerovandrei** [comment]
I tested this on devstack.
Those tests won't run on openlab, see theopenlab/openlab#365

**Comment by ozerovandrei** [comment]
recheck

**Pull request merged**